routes/apiRoutes.js

```diff
@@ -66,6 +66,11 @@ module.exports = function (app) {
     //========================================//
     app.get("/api/symbols", async (req, res) => {
         let result = await dataFetchManager.getSymbols();
+        //TEST CODE
+        result = {
+            dataSets =[],
+            dataSetsLabel =[]
+        }
```

Comment:
These lines are what we wrote as examples. these new lines shouldn't be there.

Reply:
Removed already.